### PR TITLE
Implement `sf::Rect<T>::getPosition()` and `sf::Rect<T>::getSize()`

### PR DESCRIPTION
**Pull request by ChrisThrasher**
Implements https://github.com/SFML/SFML/pull/1047

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
@dogunbound mind reviewing this?

File: src/SFML/Graphics/Rect.cpp [hunk 1]
```diff
@@ -30,6 +30,21 @@
 #include <SFML/Internal.h>
 
 
+namespace
+{
+sfVector2f toCType(const sf::Vector2f& vector)
+{
+    sfVector2f vec = {vector.x, vector.y};
+    return vec;
```
Comment (dogunbound) [inline src/SFML/Graphics/Rect.cpp:38]: Why use temporary variables here? Why not:
```c++
return {vector.x, vector.y};
```

File: src/SFML/Graphics/Rect.cpp [hunk 2]
```diff
@@ -30,6 +30,21 @@
...
+}
+
+sfVector2i toCType(const sf::Vector2i& vector)
+{
+    sfVector2i vec = {vector.x, vector.y};
+    return vec;
```
Comment (dogunbound) [inline src/SFML/Graphics/Rect.cpp:44]: Why use temporary variables here? Why not:
```c++
return {vector.x, vector.y};
```

File: src/SFML/Graphics/Rect.cpp (hunk 1 above)
Comment (ChrisThrasher) [reply src/SFML/Graphics/Rect.cpp:38]: That requires C++11. Because SFML 2 uses C++03 I'm assuming that CSFML 2 inherits that same restriction.

Comment (dogunbound) [reply src/SFML/Graphics/Rect.cpp:38]: Ah I see. Then i'll approve of this.

**Review by eXpl0it3r — APPROVED** [review]

**Pull request merged**